File: LoadingAndSaving/SavingToPdfFormat/SavingToPdfFormatExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.LoadingAndSaving.SavingToPdfFormat
	<TestClass, TestFixture> _
	Public Class SavingToPdfFormatExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub ImageToPdf()
	            TestHelper.SetDataDir("LoadingAndSaving/SavingToPdfFormat/ImageToPdf")
	
	            ImageToPdfExample.Program.Main()
        End Sub

	End Class
End Namespace

File: Examples/LoadingAndSaving/SavingToPdfFormat/SavingToPdfFormatExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.LoadingAndSaving.SavingToPdfFormat
{
    [TestClass, TestFixture]
    public class SavingToPdfFormatExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void ImageToPdf()
        {
            TestHelper.SetDataDir("LoadingAndSaving/SavingToPdfFormat/ImageToPdf");

            ImageToPdfExample.Program.Main();
        }

    }
}
